.babelrc

```diff
@@ -1,3 +1,3 @@
 {
-    "presets" : ["react"]
+    "presets" : ["es2015", "react", "stage-0"]
```

Comment:
When Babel is upgraded to version 7, Babel v7 deprecated stages (e.g. `stage-0`)
https://babeljs.io/docs/en/v7-migration#stage-preset-deprecations-https-babeljsio-blog-2018-07-27-removing-babels-stage-presets